### ISSUE TITLE
Images not copied in "dist" folder

### ISSUE THREAD

**Issue by dangelion** [WeAreAthlon/frontend-webpack-boilerplate#10]
Hi
I just noticed that images under `src/images` and `src/images/design` are not copied in the respective `dist/images` and `dist/images/design` folders.

Is this a bug or is there an explanation?

Also, I'd like to change the folders structure, that is removing folders `content` and `design` and create my own folders. I tried but it seems webpack is configured for work only with the actual structure. Right? Is it possible make it more flexible?

Another example is: removing the `favicon.ico` from `images` folder webpack returns an error. I need to remove the favicon from there, because I use another system.

Thanks a lot (your boilerplate is awesome)

**Comment by pnikolov** [comment]
Hello,
Can you post any additional information so I can try to reproduce this issue? 
For example what version are you using, steps to follow, etc.
It is recommended to follow the [ISSUE TEMPLATE](https://github.com/WeAreAthlon/frontend-webpack-boilerplate/blob/master/ISSUE_TEMPLATE.md).

Thanks

**Comment by dangelion** [comment]
Hi @pnikolov 
I just downloaded the boilerplate and installed following the instructions. Then launched `npm run watch`. Now I place an image (jpg) in `src/images/design` and it isn't copied in `dist/images/design`. I also tried launching `npm run build` but it's the same.
Notice, instead, that images in `src/images/content` are correctly copied.

Version: 3.2.0
System / OS / Browser: Chrome on Mac

Hope this could help. Thanks

**Comment by pnikolov** [comment]
Hi @dangelion,

The difference between `src/images/design` and `src/images/content` is as follows:
* **./src/images/design/** images are part of a website **design**, so they should be referenced by the SASS/CSS rules as background images. These images will be copied by `webpack` and `url-loader` see the `webpack.config.js` file.
* **./src/images/content/** images are part of a website **content** so these are mainly referenced by HTML `<img>` tags, etc. They are copied by the `CopyWebpackPlugin`, you can review the `webpack.config.js` in the part where this plugin is configured:
```js
new CopyWebpackPlugin([
  {
    from: path.resolve(__dirname, 'src', 'images', 'content'),
    to: path.resolve(__dirname, 'dist', 'images', 'content'),
    toType: 'dir',
  },
]),
```

So, to summarise - **no** images placed under the `./src/images/design` directory will be copied if **they are not referenced** from your _SASS_ files - that is why you don't see them in your `dist` output directory.
